bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/handler/NetworkHandler.java

```diff
@@ -146,10 +149,10 @@ public void finalDetectionResult(PresenceDetectionValue value) {
             retryCounter = 0;
         }
 
-        if (value.isReachable()) {
-            Instant instant = Instant.ofEpochMilli(presenceDetection.getLastSeen());
+        Instant lastSeen = presenceDetection.getLastSeen();
+        if (value.isReachable() && lastSeen != null) {
```

Comment:
Should it be skipped or updated to **UNDEF** if `lastSeen` is null?

Reply:
I added UNDEF in case the device is unreachable and lastSeen is `null`.